lib/rules/no-invalid-attribute-name.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @author Doug Wade <douglas.b.wade@gmail.com>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+const xnv = require('xml-name-validator')
```

Comment:
This line has a red squiggly in my text editor because there aren't types for it so I [opened a PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59989) to add some